SofaKernel/modules/SofaSimulationTree/ExportDotVisitor.h

```diff
@@ -19,93 +19,4 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#ifndef SOFA_SIMULATION_TREE_EXPORTDOTACTION_H
-#define SOFA_SIMULATION_TREE_EXPORTDOTACTION_H
-
-#include <SofaSimulationTree/GNodeVisitor.h>
-#include <string>
-#include <iostream>
-
-
-namespace sofa
-{
-
-namespace simulation
-{
-
-namespace tree
-{
-
-class SOFA_SIMULATION_TREE_API ExportDotVisitor : public GNodeVisitor
-{
-public:
-    std::ostream* out;
-
-    bool showNode;
-    bool showObject;
-    bool showBehaviorModel;
-    bool showCollisionModel;
-    bool showVisualModel;
-    bool showMapping;
-    bool showContext;
-    bool showCollisionPipeline;
-    bool showSolver;
-    bool showMechanicalState;
-    bool showForceField;
-    bool showInteractionForceField;
-    bool showConstraint;
-    bool showMass;
-    bool showTopology;
-    bool showMechanicalMapping;
-
-    bool labelNodeName;
-    bool labelNodeClass;
-    bool labelObjectName;
-    bool labelObjectClass;
-
-    ExportDotVisitor(const sofa::core::ExecParams* params, std::ostream* out);
-    ~ExportDotVisitor() override;
-
-    void processObject(GNode* node, core::objectmodel::BaseObject* obj);
-
-    Result processNodeTopDown(GNode* node) override;
-    void processNodeBottomUp(GNode* node) override;
-
-    const char* getClassName() const override { return "ExportDotVisitor"; }
-
-protected:
-
-    /// None names in output
-    std::map<core::objectmodel::Base*, std::string> names;
-    /// Next indice available for duplicated names
-    std::map<std::string, int> nextIndex;
-
-    /// Test if a node should be displayed
-    bool display(GNode* node, const char** color=nullptr);
-
-    /// Test if an object should be displayed
-    bool display(core::objectmodel::BaseObject* obj, const char** color=nullptr);
-
-    /// Find the node or object a given object should be attached to.
-    /// This is the parent node if it is displayed, otherwise it is the attached MechanicalState or Solver.
-    /// Returns an empty string if not found.
-    std::string getParentName(core::objectmodel::BaseObject* obj);
-
-    /// Compute the name of a given node or object
-    std::string getName(core::objectmodel::Base* o, std::string prefix);
-
-    /// Compute the name of a given node
-    std::string getName(core::objectmodel::BaseNode *node);
-
-    /// Compute the name of a given object
-    std::string getName(core::objectmodel::BaseObject* obj);
-
-};
-
-} // namespace tree
-
-} // namespace simulation
-
-} // namespace sofa
-
-#endif
+#error "SofaSimulationTree was removed at release 21.06. You need to update your code by replacing #include<SofaSimulationTree/ExportDotVisitor.h> with #include<sofa/simulation/ExportDotVisitor.h>
```

Comment:
No end of string? (no double quotes at the end)
(same for the other #error message)

Reply:
Good eyes, 
as there is no need for quotes at all (https://en.cppreference.com/w/c/preprocessor/error) I will remove them.